### ISSUE TITLE
Testing: Swapfile size increase to 10GB

### ISSUE THREAD

**Issue by rootzoll** [raspiblitz/raspiblitz#4131]
Implmented in #4114 
Check if a recover will increase the swapfile after recover `/mnt/hdd/swapfile`

**Comment by rootzoll** [comment]
looks good on recover
```
ls -la /mnt/hdd/swapfile
-rw------- 1 root root 8082423808 Aug 25 20:28 /mnt/hdd/swapfile
```

**Comment by rootzoll** [comment]
Found a node where after update its still at:
```
-rw------- 1 root root 4003463168 Sep 13 13:35 /mnt/hdd/swapfile
```

Push to v1.10.1 for further investigation

**Comment by rootzoll** [comment]
OK dphys-swapfile might change the size of the swap file by it self - we just changed the "CONF_MAXSWAP" so its just the upper limited, not the forced size.

Pushing to v1.10.1 for further obervation.
And teamdiscuss to maybe force a fixed swap size with a v1.11 version with `CONF_SWAPSIZE=10240` value in `/etc/dphys-swapfile`.

**Comment by rootzoll** [comment]
Under Bookworm and RP5 a fresh installed RaspiBlitz shows already the almost 10GB SwapSize ... so I think were good and this issue can be closed for now.

```
admin@192.168.178.182:~ ₿ lsblk
NAME        MAJ:MIN RM   SIZE RO TYPE MOUNTPOINTS
sda           8:0    0 931.5G  0 disk
└─sda1        8:1    0 931.5G  0 part /mnt/hdd
mmcblk0     179:0    0  29.8G  0 disk
├─mmcblk0p1 179:1    0   512M  0 part /boot/firmware
└─mmcblk0p2 179:2    0  29.3G  0 part /
zram0       253:0    0   9.8G  0 disk [SWAP]
```
